### DISCUSSION TITLE
Export the property data with photos and texts

### DISCUSSION THREAD

**Discussion by Enr62** [JoomlaEstateAgency/com_jea#36] [Q&A] [answered]
Hi, on JEA I can export the setting, but I can't export the property data with photos and texts. I would like to take them from a Joomla 3 site and insert them into a new Joomla 4 site (avoiding inserting them all again, one at a time).
Thanks for the tips

**Comment by ilhooq** [comment, marked as answer]
Hi @Enr62 ,

From your Joomla 3 site, in the administrator area, go to components > JEA > tools then click on export.

On the export page, click on the gateways button then create a new gateway and use JEA as provider.

Return to the export page, then click on the `Start Export` button in the Ajax tab. You should have a zip to download.

From your Joomla 4 site, reproduce these steps but using the import tool.

**Comment by Enr62** [comment]
Thanks for the reply, and where could I find the zip folder with the data (to import onto the new site?)

**Reply by ilhooq** [reply]
When you will create a new import gateway in the new site, you'll specify a path where to find your zip file.

**Comment by Enr62** [comment]
I tested it on a another test site with only one property, and it gave me the option to download the zip folder to the desktop. Probably on the site with many properties it takes longer, but it has been running for more than 20 hours!!! There must be something wrong...

**Comment by Enr62** [comment]
Unfortunately I can't find clear explanations to read to do these things (even paying), avoiding asking for help. It's a shame, because JEA is a nice extension for real estate agencies, but if, for example, there is no way to see how and where the properties are exported, changing the site becomes unusable and makes you want to abandon it!
Thanks anyway for the help received.
Merry Christmas

**Comment by ilhooq** [comment]
Another way :
To transfer JEA tables to a new database, you can utilize Phpmyadmin to export the tables as a dump file and then import it into the new database. As for the pictures, you can simply copy the repository images/com_jea from the original location to the new location.

**Reply by Enr62** [reply]
Thank you so much for your help! By operating in this way the importation of properties is effectively solved! Thanks again!
It still remains to be understood why the "SLOGANS" appear wherever they want, in strange positions on the page or on the photos in JEA Slide (which in turn should not be seen in the search results...)
The settings in the backend are correct. What could it depend on?
![SCR-20240103-nrhh](https://github.com/JoomlaEstateAgency/com_jea/assets/140328450/8cc166f8-f860-4298-85ae-ba9e19d13c1b)
![SCR-20240103-pkvu](https://github.com/JoomlaEstateAgency/com_jea/assets/140328450/9c94ff8e-5488-4c22-8b5a-b263c2a80759)

**Comment by Enr62** [comment]
Okay. I corrected the incorrect position of the SLOGANS, it was caused by a position error in the CSS (from "ABSOLUTE" to "RELATIVE"), it remains to understand how not to make it appear on the SLIDER, even better, the SLIDER should also not be present in the results of searches, nor the SEARCH module. I don't know PHP and I struggle a bit to solve these things, but I'm trying!
